spec/lib/locasms/client_spec.rb

```diff
@@ -4,6 +4,40 @@
   let(:rest_client) { 'rest_client mock' }
   subject { LocaSMS::Client.new :login, :password, rest_client: rest_client }
 
+  context '.ENDPOINT' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.